src/open_clip/factory.py

```diff
@@ -317,6 +320,12 @@ def _convert_ln(m):
         force_preprocess_cfg['size'] = model.visual.image_size
     set_model_preprocess_cfg(model, merge_preprocess_dict(preprocess_cfg, force_preprocess_cfg))
 
+    if add_adapters:
+        adapters.init(model.text.transformer)
+        adapter_config = AdapterConfig.load("seq_bn")
+        model.text.transformer.add_adapter("test_adapter", adapter_config)
```

Comment:
So the model from OpenCLIP already has a builtin method called `add_adapter`?

Reply:
No, it's from the [PEFT](https://github.com/huggingface/peft) library, that adds support for this stuff to `transformers`.